lib/src/widgets/chat/ds_interactive_voice_call_message_bubble.widget.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+
+import '../../enums/ds_align.enum.dart';
+import '../../enums/ds_border_radius.enum.dart';
+import '../../models/ds_message_bubble_style.model.dart';
+import '../../models/interactive_message/ds_interactive_message.model.dart';
+import '../../themes/colors/ds_colors.theme.dart';
+import '../../themes/icons/ds_icons.dart';
+import '../texts/ds_body_text.widget.dart';
+import '../utils/ds_divider.widget.dart';
+import 'ds_message_bubble.widget.dart';
+
+class DSInteractiveVoiceCallMessageBubble extends StatefulWidget {
```

Comment:
This widget could be a `StatelessWidget`.

Reply:
Done!